lib/licensed/sources/gradle.rb

```diff
@@ -84,16 +91,23 @@ def enabled?
       end
 
       def enumerate_dependencies
-        JSON.parse(self.class.gradle_command("printDependencies", path: config.pwd, executable: gradle_executable, configurations: configurations)).map do |package|
-          name = "#{package["group"]}:#{package["name"]}"
+        JSON.parse(self.class.gradle_command(
+          "printDependencies",
+          path: config.pwd.join,
+          executable: gradle_executable,
+          configurations: configurations,
+          project: project
+        )).map do |package|
+          name = "#{package['group']}:#{package['name']}"
           Dependency.new(
             name: name,
             version: package["version"],
             path: config.pwd,
             executable: gradle_executable,
             configurations: configurations,
+            project: project,
```

Comment:
is it possible for different projects in a multi-project build to have different versions of the same dependency? 

If yes, this value should be part of the `name:` argument rather than being added as a new property value.